### DISCUSSION TITLE
membership incentives

### DISCUSSION THREAD

**Discussion by tiMaxal** [RunDavidMC/IoniFaucet#12] [Ideas]
ideas to draw users to make an account:  
  
- convenience of gifting [not having to find the Name in NB-dash 'domains' and then manually create the gift with the giftee's email address]
- personalized landing page link for gifting [ie, members' own business-oriented page]
- faucet-points incentives
    - gain rank for, ie .. use of generic donated names if own run out, n still get NB-referral bonus ..?
    - permission for gifting SLDs on Faucet TLDs n still get NB-referral bonus ..?

**Comment by RunDavidMC** [comment]
Sounds good. We could even allow custom discord embeds for user pages.
